tools/unit_tests.py

```diff
@@ -45,14 +45,16 @@ def run_unit_test(deno_exe, permStr, flags=None):
 # tests by the special string. permW0N0 means allow-write but not allow-net.
 # See js/test_util.ts for more details.
 def unit_tests(deno_exe):
-    run_unit_test(deno_exe, "permR0W0N0E0U0", ["--reload"])
-    run_unit_test(deno_exe, "permR1W0N0E0U0", ["--allow-read"])
-    run_unit_test(deno_exe, "permR0W1N0E0U0", ["--allow-write"])
-    run_unit_test(deno_exe, "permR1W1N0E0U0",
+    run_unit_test(deno_exe, "permR0W0N0E0U0H0", ["--reload"])
+    run_unit_test(deno_exe, "permR1W0N0E0U0H0", ["--allow-read"])
```

Comment:
As an aside (for another time), I feel like `r-----` `rw----` etc would be a little clearer/more concise here.

Reply:
Well, this whole system needs to be refactored actually. Currently it's possible there are some unit tests that don't get run because they don't match one of the listed patterns.

Reply:
I can refactor this in another PR if needed.